pkg/vm/context.go

```diff
@@ -226,7 +226,11 @@ func (c *Context) String() string {
 // GetContextScriptHash returns script hash of the invocation stack element
 // number n.
 func (v *VM) GetContextScriptHash(n int) util.Uint160 {
-	ctxIface := v.Istack().Peek(n).Value()
+	element := v.Istack().Peek(n)
+	if element == nil {
+		return util.Uint160{}
+	}
+	ctxIface := element.Value()
```

Comment:
Not sure that this is the right plays for such check. Should we check it in calling methods?

Reply:
I think, panic is ok here. I see no situations, where we need some script hash without knowing if it exists.
In what circumstances a panic occurs in your case?

Reply:
E.g. in `CheckHashedWitness(...)` when we'll try to get `callingScriptHash`. It could be that we have only 1 item at Istack.